DscResources/MSFT_xMySqlDatabase/MSFT_xMySqlDatabase.psm1

```diff
@@ -51,120 +8,119 @@ function Get-TargetResource
     param
     (
         [parameter(Mandatory = $true)]
-        [ValidateSet("Present", "Absent")]
-        [string] $Ensure = "Present",
-
-        [parameter(Mandatory = $true)]
-        [string] $Name,
+        [ValidateNotNullOrEmpty()]
+        [string] $DatabaseName,
         
         [parameter(Mandatory = $true)]
-        [pscredential] $ConnectionCredential
+        [pscredential] $RootCredential,
+
+        [parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string] $MySqlVersion
     )
     
-    $Ensure = "Absent"
+    if (Test-Path $ErrorPath)
+    {
+        Remove-Item -Path $ErrorPath
+    }
+  
+    $arguments = "--execute=SELECT IF(EXISTS (SELECT SCHEMA_NAME FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME = '$DatabaseName'), 'Yes','No')", `
+        "--user=root", "--password=$($RootCredential.GetNetworkCredential().Password)", "--port=$(Get-MySqlPort -MySqlVersion $MySqlVersion)", "--silent"
```

Comment:
Thanks. Should not there be an include that imports Utilities helper in the module psm1?